File: algoritmos/Eje-3/9/9.py
def calc_incremento(recarga):
  importe = 0
  if recarga >= 500 and recarga < 700: return recarga * .2
  if recarga >= 700 and recarga < 900: return recarga * .5
  if recarga >= 900: return recarga * 1
  return importe

def info_recarga(nro,recarga):
  incremento = calc_incremento(recarga)
  print(f"Se realizó una recarga de ${recarga} al número {nro}")
  print(f"El total a pagar es ${recarga + incremento}")

nro = input("Ingrese el número de celular que desea recargar")
recarga = int(input("Ingrese el monto de la recarga"))
info_recarga(nro,recarga)

File: algoritmos/Eje-3/9/9.c
#include <stdio.h>
#include <stdlib.h>

float calcIncremento(float valor){
  float importe = 0;
  if(valor >= 500)
    importe = valor * .2;
  if(valor >= 700)
    importe = valor * .5;
  if(valor >= 900)
    importe = valor * 1;
  return importe;
}

void infoRecarga(char nro[10], float recarga){
  float incremento = calcIncremento(recarga);
  printf("Se realizo una recarga de $%.2f al numero %s",recarga, nro);
  printf("\nEl total a pagar es $%.2f", recarga + incremento);
  return;
}

int main(){
  char nro[10];
  float recarga;
  printf("Ingrese el numero de celular que desea recargar\n");
  fflush(stdin);
  scanf("%s",&nro);
  printf("INgrese el monto de la recarga\n");
  fflush(stdin);
  scanf("%f",&recarga);
  infoRecarga(nro,recarga);
}